source/client-side-encryption/tests/localSchema.json

```diff
@@ -0,0 +1,276 @@
+                            },
+                            "$db": "admin"
+                        },
+                        "command_name": "find"
+                    }
+                },
+                {
+                    "command_started_event": {
+                        "command": {
+                            "insert": "default",
+                            "documents": [
+                                {
+                                    "_id": 1,
+                                    "encrypted_string": {
+                                        "$binary": {
+                                            "base64": "AQAAAAAAAAAAAAAAAAAAAAACtsdOjHK3CkpNjAAMznkYbeR6Z+yLzCRv4zOb7VfKnJdmU0W5MD8GVODx8K+KuBoCbE1SfGfPgX6NhS/RKpQ12w==",
+                                            "subType": "06"
+                                        }
+                                    }
+                                }
+                            ],
+                            "ordered": true
+                        },
+                        "command_name": "insert"
+                    }
+                },
+                {
+                    "command_started_event": {
+                        "command": {
+                            "find": "default",
+                            "filter": {
+                                "_id": 1
+                            }
+                        },
+                        "command_name": "find"
+                    }
+                }
+            ],
+            "outcome": {
+                "collection": {
+                    "data": [
+                        {
+                            "_id": 1,
+                            "encrypted_string": {
+                                "$binary": {
+                                    "base64": "AQAAAAAAAAAAAAAAAAAAAAACtsdOjHK3CkpNjAAMznkYbeR6Z+yLzCRv4zOb7VfKnJdmU0W5MD8GVODx8K+KuBoCbE1SfGfPgX6NhS/RKpQ12w==",
+                                    "subType": "06"
+                                }
+                            }
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "description": "A local schema with no encryption is an error",
+            "clientOptions": {
+                "autoEncryptOpts": {
+                    "schemaMap": {
+                        "default.default": {
+                            "properties": {
+                                "test": {
+                                    "bsonType": "string"
+                                }
+                            },
+                            "bsonType": "object",
+                            "required": [
+                                "test"
+                            ]
+                        }
+                    },
+                    "kmsProviders": {
+                        "aws": {}
+                    }
+                }
+            },
+            "operations": [
```

Comment:
This definitely doesn't pass on j3071 branch. It should be just:

           "operations": [
                {
                    "name": "insertOne",
                    "arguments": {
                        "document": {
                            "_id": 1,
                            "encrypted_string": "string0"
                        }
                    },
                    "result": {
                        "errorContains": "JSON schema keyword 'required' is only allowed with a remote schema"
                    }
                }

Reply:
gah I updated the template file but forgot to regenerate the .json/.yml files. Fixed.